src/context/PluginProvider/activePlugins.ts

```diff
@@ -0,0 +1,19 @@
+import avalanche from 'plugins/avalanche'
+import bitcoin from 'plugins/bitcoin'
+import cosmos from 'plugins/cosmos'
+import dogecoin from 'plugins/dogecoin'
+import ethereum from 'plugins/ethereum'
+import foxPage from 'plugins/foxPage'
+import osmosis from 'plugins/osmosis'
+import pendo from 'plugins/pendo'
+
+export const activePlugins = [
+  bitcoin,
+  cosmos,
+  dogecoin,
+  ethereum,
+  foxPage,
+  osmosis,
+  avalanche,
+  pendo,
+]
```

Comment:
It may be better to move this file to `src/plugins` instead.

Reply:
want to tackle in said future PR?

Reply:
Moved in https://github.com/shapeshift/web/pull/2209/commits/75cedd3ff8f5d590242caaa6a1a4e3f09b1244b8